SSO-Auth/Api/SSOController.cs

```diff
@@ -787,7 +808,7 @@ private async Task<Guid> CreateCanonicalLinkAndUserIfNotExist(string mode, strin
             UpdateCanonicalLinkConfig(links, mode, provider);
         }
 
-        Guid userId = Guid.Empty;
+        userId = Guid.Empty;
         try
```

Comment:
Doesn't your code duplicate the code down here?

Reply:
You can kinda look at it as a verification of the user creation being done successfully if that makes sense haha.

Reply:
Makes sense, thanks for the clarification!